### DISCUSSION TITLE
Progress bar Text

### DISCUSSION THREAD

**Discussion by HeRm4nV** [jspsych/jsPsych#1253] [Q&A] [answered]
Is there a way to change the text in the progress bar? ("Completion Progress"), i'm doing a instrument in spanish, so this would be quite useful

**Comment by jodeleeuw** [comment, marked as answer]
Yes, you can set the `message_progress_bar` parameter in `jsPsych.init()`

```js
jsPsych.init({
	timeline: [...],
	message_progress_bar: 'Custom Message Here'
});
```

**Reply by HeRm4nV** [reply]
thank's for your quick reply, I updated the jspsych.js and it works!
